docs/strategy-advanced.md

```diff
@@ -42,32 +42,38 @@ class AwesomeStrategy(IStrategy):
 
 ***
 
-### Storing custom information using DatetimeIndex from `dataframe`
+## Custom sell signal
 
-Imagine you need to store an indicator like `ATR` or `RSI` into `custom_info`. To use this in a meaningful way, you will not only need the raw data of the indicator, but probably also need to keep the right timestamps.
+It is possible to define custom sell signals. This is very useful when we need to customize sell conditions for each individual trade, or if you need the trade profit to take the sell decision.
 
-```python
-import talib.abstract as ta
-class AwesomeStrategy(IStrategy):
-    # Create custom dictionary
-    custom_info = {}
-
-    def populate_indicators(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
-        # using "ATR" here as example
-        dataframe['atr'] = ta.ATR(dataframe)
-        if self.dp.runmode.value in ('backtest', 'hyperopt'):
-          # add indicator mapped to correct DatetimeIndex to custom_info
-          self.custom_info[metadata['pair']] = dataframe[['date', 'atr']].set_index('date')
-        return dataframe
-```
+An example of how we can use different indicators depending on the current profit and also sell trades that were open longer than 1 day:
 
-!!! Warning
-    The data is not persisted after a bot-restart (or config-reload). Also, the amount of data should be kept smallish (no DataFrames and such), otherwise the bot will start to consume a lot of memory and eventually run out of memory and crash.
+``` python
+from freqtrade.strategy import IStrategy, timeframe_to_prev_date
 
-!!! Note
-    If the data is pair-specific, make sure to use pair as one of the keys in the dictionary.
+class AwesomeStrategy(IStrategy):
+    def custom_sell(self, pair: str, trade: 'Trade', current_time: 'datetime', current_rate: float,
+                    current_profit: float, dataframe: DataFrame, **kwargs):
+        trade_open_date = timeframe_to_prev_date(self.timeframe, trade.open_date_utc)
+        trade_row = dataframe.loc[dataframe['date'] == trade_open_date].squeeze()
+
+        # Sell when price falls below value in stoploss column of taken buy signal.
+        # above 20% profit, sell when rsi < 80
+        if current_profit > 0.2:
+            if trade_row['rsi'] < 80:
+                return 'rsi_below_80'
+
+        # Between 2% and 10%, sell if EMA-long above EMA-short
+        if 0.02 < current_profit < 0.1:
+            if trade_row['emalong'] > trade_row['emashort']:
+                return 'ema_long_below_80'
+
+        # Sell any positions at a loss if they are held for more than two days.
+        if current_profit < 0.0 and (current_time - trade.open_date_utc).days >= 1:
```

Comment:
We missed a comment. When you get to it, could you please fix it to say "more than one day"?

Reply:
aaaah, nice catch 👍 
addressed in 5bc908870f124f27fe16f890b895da8af970c2f9